src/test/groovy/graphql/nadel/hooks/ServiceExecutionHooksTest.groovy

```diff
@@ -171,9 +173,11 @@ class ServiceExecutionHooksTest extends Specification {
         }
         """)
 
-        def overallSchema = TestUtil.schema("""
-        type Query {
-            foo(id: String): String
+        def overallSchema = TestUtil.schemaFromNdsl("""
+        service service1 {
+            type Query {
+                foo(id: String): String
+            }
```

Comment:
these changes to the tests are needed in order for the schema definitions to have field ids in the test data